resources/aws/vpc.go

```diff
@@ -0,0 +1,117 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	microerror "github.com/giantswarm/microkit/error"
+	// "github.com/giantswarm/aws-operator/resources"
+)
+
+const (
+	cidrBlock         = "10.0.0.0/16"
```

Comment:
Wouldn't it be better to have an issue instead? That seems to be the common practice so far.